cmd/initialize/init.go

```diff
@@ -0,0 +1,31 @@
+// Copyright 2022 Red Hat, Inc.
```

Comment:
`// Copyright The Enterprise Contract Contributors`

Check other files as well.